src/lib/core/fiber_channel.c

```diff
@@ -58,6 +59,13 @@ struct ipc_wait_pad {
 static enum fiber_channel_close_mode fiber_channel_close_mode =
 	FIBER_CHANNEL_CLOSE_FORCEFUL;
 
+static const char *const fiber_channel_close_mode_strs[] = {
+	"forceful",
+	"graceful",
+};
+
+TWEAK_ENUM(fiber_channel_close_mode, fiber_channel_close_mode);
+
 void
 fiber_channel_set_close_mode(enum fiber_channel_close_mode mode)
```

Comment:
> Note that we don't drop fiber_channel_set_close_mode function, because
> it's still used in unit tests.

Shouldn't tweaks be easily accessible from unit tests? Or I missed some point why the function should be used in the unit test and not a tweak setter?

Reply:
Yes, tweaks are accessible from unit tests, but I don't want to refactor the test in the scope of this commit (note, I said "we **don't** drop", not "we **can't** drop"). If we decide to drop `fiber_channel_set_close_mode`, we have two options:
1. Use the tweaks API in the test (`tweak_find`, `tweak_set`). This would look cumbersome IMO, taking into account the fact that we just want to toggle the value of a C variable from C code, i.e. we could just as well toggle it directly, which brings us to the second option.
2. Move the `fiber_channel_close_mode` variable to the header and toggle it directly in the test. This would bloat the patch with an unrelated changed.

IMO the code looks fine as it is so I decided to leave it as is.

Reply:
Okay, thanks for the explanation.